doc/_static/custom_scripts.js

```diff
@@ -0,0 +1,162 @@
+            },
+          },
+          {
+            element: "#toolbar-rect",
+            popover: {
+              title: "Toolbar",
+              description: 'The toolbar allows you to quickly access some of the most common commands in Spyder.',
+              position: "bottom",
+            },
+          },
+          {
+            element: "#statusbar-rect",
+            popover: {
+              title: "Status Bar",
+              description: 'The status bar shows your current Python environment, git branch, memory usage and various attributes of the currently active file.',
+              position: "top",
+            },
+          },
+          {
+            element: "#options-menu-rect",
+            popover: {
+              title: "Options Menu",
+              description: 'You can display the options menu by clicking the "hamburger" icon at the top right of the pane. It contains useful settings and actions relevant to the pane.',
+              position: "right",
+            },
+          },
+          {
+            element: "#context-menu-rect",
+            popover: {
+              title: "Context Menu",
+              description: 'To display the context menu for a pane, right-click anywhere over it. The menu shows actions relevant to the element you are hovering over.',
+              position: "right",
+            },
+          },
+          {
+            element: "#editor-rect",
+            popover: {
+              title: "Editor",
+              description: 'The <a href="editor.html">Editor</a> is the pane where you can create, open and edit files. It contains useful features like autocompletion, real-time analysis and syntax highlighting.',
+              position: "right",
+            },
+          },
+          {
+            element: "#console-rect",
+            popover: {
+              title: "IPython Console",
+              description: 'The <a href="ipythonconsole.html">Console</a> allows you to run your code from the Editor or interactively. You can also use it to control Spyder’s debugger.',
+              position: "left",
+            },
+          },
+          {
+            element: "#help-rect",
+            popover: {
+              title: "Help",
+              description: 'The <a href="help.html">Help</a> pane displays documentation for the objects you are using in the Editor or the IPython Console. To trigger Help, press Ctrl-I (Cmd-I on macOS) with your cursor over an object, or type its name in the object field.',
+              position: "left",
+            },
+          },
+          {
+            element: "#variable-explorer-rect",
+            popover: {
+              title: "Variable Explorer",
+              description: 'The <a href="variableexplorer.html">Variable Explorer</a> allows you to browse and interact with the objects generated running your code. Double-clicking a variable will open a specialized viewer, allowing you to inspect its contents.',
```

Comment:
```suggestion
              description: 'The <a href="variableexplorer.html">Variable Explorer</a> allows you to browse and interact with the objects generated after running your code. Double-clicking a variable will open a specialized viewer, allowing you to inspect its contents.',
```

That `after` feels missing to me as a non-native speaker.